doc/releases/release-notes-3.1.rst

```diff
@@ -87,6 +87,9 @@ Bluetooth
 
   * The :c:macro:bt_conn_index function now takes a `const struct bt_conn`.
 
+  * The `struct bt_gatt_subscribe_params` :c:func:`write` callback
+    function has been replaced with a :c:func:`subscribe` callback
+    function.
```

Comment:
If we are deprecating the `write` instead of replacing it, this will need to be updated

Reply:
Why?
The write has been replaced by a subscribe, and one is supposed to use the new one.
(It could of course say that the write has been deprecated as well.)

Reply:
Because if it is deprecated, then we did not replace it, but rather `deprecated :c:func:`write` and `added :c:func:`subscribe`. Since both still exist, it has not yet been "replaced".

Reply:
Changed